python/python/glide/async_commands/sorted_set.py

```diff
@@ -23,6 +23,23 @@ class InfBound(Enum):
     """
 
 
+class ScoreFilter(Enum):
+    """
+    Defines which elements to pop from a sorted set.
+
+    ScoreFilter is a mandatory option for BZMPOP (https://valkey.io/commands/bzmpop).
```

Comment:
```suggestion
    ScoreFilter is a mandatory option for [BZMPOP](https://valkey.io/commands/bzmpop).
```

Reply:
Same as above, if its alright with you I'll leave this as is until we've chatted with Bar and settled on a decision